modules/ossm-federation-limitations.adoc

```diff
@@ -0,0 +1,13 @@
+////
+This module included in the following assemblies:
+* service_mesh/v2x/ossm-federation.adoc
+////
+
+[id="ossm-federation-limitations_{context}"]
+= Federation limitations
+
+The {ProductName} federated approach to joining meshes has the following limitations:
```

Comment:
At some point someone mentioned them and I wrote this down in my notes.  It might have been a testing limitation?  I figured better to put them in for now and have someone tell me I was wrong.    ¯\_(ツ)_/¯

Reply:
Good on you :) I guess we never tested them, so this might be correct. Maybe we just note that these scenarios haven't been tested?